### DISCUSSION TITLE
CORSAIR Hydro Series H60 (CW-9060007-WW)

### DISCUSSION THREAD

**Discussion by galtline** [liquidctl/liquidctl#423] [General]
I see liquidctl has support for H60i but not sure if it will work with H60. When I try to find the hardware liquidctl cannot locate it. 

I am not sure if the hydro series comes with software. I think you are suppose to just plug the fan into the CPU port on the board.

**Comment by jonasmalacofilho** [comment]
You're right, it doesn't (see #131).

There is no (high-level) controller in this new cooler model.  This means that it isn't a device liquidctl can support directly.
